bond_test.go

```diff
@@ -25,6 +26,12 @@ type database struct {
 
 	Account AccountStore `collection:"accounts"`
 	User    UserStore    `collection:"users"`
+	Log     LogStore     `collection:"logs"`
+}
+
+type Log struct {
```

Comment:
This is a new struct I added in order to test transactions. Each time an user or an account is inserted, a log will be added to this table.